test/integration/IntegrationTestListener.php

```diff
@@ -0,0 +1,87 @@
+<?php
+/**
+ * Zend Framework (http://framework.zend.com/)
+ *
+ * @link      http://github.com/zendframework/zf2 for the canonical source repository
+ * @copyright Copyright (c) 2005-2016 Zend Technologies USA Inc. (http://www.zend.com)
+ * @license   http://framework.zend.com/license/new-bsd New BSD License
+ */
+
+namespace ZendIntegrationTest\Db;
+
+use Exception;
+use PDO;
+use PDOException;
+use PHPUnit\Framework\BaseTestListener;
+use PHPUnit_Framework_TestSuite as TestSuite;
+
+class IntegrationTestListener extends BaseTestListener
+{
+    /**
+     * @var PDO
+     */
+    private $pdo;
+
+    public function startTestSuite(TestSuite $suite)
+    {
+        if ($suite->getName() !== 'integration test') {
+            return;
+        }
+        printf("\nIntegration test started.\n");
+
+        if (getenv('TESTS_ZEND_DB_ADAPTER_DRIVER_MYSQL')) {
+            $this->createMysqlDatabase(__DIR__ . '/TestAsset/mysql.sql');
+        }
+    }
+
+    public function endTestSuite(TestSuite $suite)
+    {
+        if ($suite->getName() !== 'integration test') {
+            return;
+        }
+        printf("\nIntegration test ended.\n");
+
+        if (getenv('TESTS_ZEND_DB_ADAPTER_DRIVER_MYSQL')) {
+            $this->dropMysqlDatabase();
+        }
+    }
+
+    private function createMysqlDatabase($dbFile)
```

Comment:
Keep to convention used around the codebase of having this in _Platform_ class. Will make possible adding other engines.